ui/component/router/view.jsx

```diff
@@ -302,15 +308,18 @@ function AppRouter(props: Props) {
   }, [hasDefaultChannel]);
 
   React.useEffect(() => {
-    // has a default channel selected, clear the current active channel
-    if (
+    if (window.pendingActiveChannel) {
+      doSetActiveChannel(window.pendingActiveChannel);
+      delete window.pendingActiveChannel;
+    } else if (
```

Comment:
https://github.com/OdyseeTeam/odysee-frontend/pull/2147/commits/26b98082510b94a6d0d784d19edf6467a4d2fa6b When you are on a channel that you own but have a different active channel -> "create tiers for this channel" brings you to this channel's membership overview, I went with window since it's just for a "pending" value to be set when a new page is hit and then deleted